wadas/domain/animal_detection_mode.py

```diff
@@ -46,11 +47,44 @@ def run(self):
                 ):
                     logger.info("Instantiating FTPS server...")
                     self.ftp_thread = FTPsServer.ftps_server.run()
+        logger.info("Ready for video stream from Camera(s)...")
 
-        self.start_actuator_server()
+    def _detect(self, cur_image_path):
```

Comment:
I would say that _detect() method could be moved to generic class operation mode, as it is intent of WADAS to perform at least the animal detection.